Readme - Minecraft Edu Updates (#185)
microsoft/StudentsAtBuild: README.md

| COM235 | Thanks for joining the Student Zone at Build | [Chloe Condon](https://twitter.com/ChloeCondon), [Crazy Aunt Lindsey](https://www.thefablab.com/), [Liam McGregor](www.linkedin.com/in/liamjmcgregor/), [Morgan Dukes](https://www.linkedin.com/in/morgandukes3/), [Swetha Prabakaran](twitter.com/swethapraba) | [Microsoft the Musical](http://aka.ms/musical) |  |  |

## Try out Minecraft: Education Edition with MakeCode

You will need an Office 365 Education account to log in to the full version of Minecraft: Education Edition. If you do not have a license, you can still use Minecraft: Education Edition with a limited number of logins.

Enter your [school email address](https://education.minecraft.net/get-started/) to see if your school account is eligible for a trial. If it’s not, we’ll redirect you to a sign-up page.

### You can try out the Minecraft: Education Edition with these usernames:  
mee01@minecrafteducationedition.com  
mee02@minecrafteducationedition.com  
mee03@minecrafteducationedition.com  
...  
mee99@minecrafteducationedition.com  
Pick one, doesn't matter

> Password for all: Build2020  

Reviewer: ```suggestion
Password for all: Build2020  
  
**NOTE: THESE USER ACCOUNTS MAY NOT BE AVAILABLE PAST BUILD. HOWEVER, ANY STUDENT WITH A M365 SCHOOLS ACCOUNT WILL BE ABLE TO LOG INTO MINECRAFT EDUCATION**
```

### Minecraft Education 

> -You can find more details of [Minecraft education](https://education.minecraft.net/)

Reviewer 2: Here I would refer to it as Minecraft: Education Edition as was done above

Reviewer: Agreed, Thanks @salmanmkc !

> -You can find more details of [Minecraft education](https://education.minecraft.net/)

Reviewer: ```suggestion
-You can find more details of [Minecraft: Education Edition](https://education.minecraft.net/)
```

> -Install [Minecraft Education](https://education.minecraft.net/get-started/)

Reviewer 2: Same as last, refering to as Minecraft: Education Edition

> -Install [Minecraft Education](https://education.minecraft.net/get-started/)

Reviewer: ```suggestion
-Install [Minecraft: Education Edition](https://education.minecraft.net/get-started/)
```

> 

Reviewer: ```suggestion
  
### Educational Content on Minecraft
```

## Try out Minecraft: Education Edition with MakeCode

You will need an Office 365 Education account to log in to the full version of Minecraft: Education Edition. If you do not have a license, you can still use Minecraft: Education Edition with a limited number of logins.

Enter your [school email address](https://education.minecraft.net/get-started/) to see if your school account is eligible for a trial. If it’s not, we’ll redirect you to a sign-up page.

### You can try out the Minecraft: Education Edition with these usernames:  
mee01@minecrafteducationedition.com  
mee02@minecrafteducationedition.com  
mee03@minecrafteducationedition.com  
...  
mee99@minecrafteducationedition.com  
Pick one, doesn't matter
Password for all: Build2020  

### Minecraft Education 

-You can find more details of [Minecraft: Education Edition](https://education.minecraft.net/)
-Install [Minecraft: Education Edition](https://education.minecraft.net/get-started/)
  
### Educational Content on Minecraft

> Minecraft has released a series of free educational sessions that parents can download to help children learn at home during COVID-19

Reviewer: ```suggestion
[Minecraft](https://www.minecraft.net/) has released a series of free educational sessions that parents can download to help children learn at home during COVID-19.
```

> Minecraft Marketplace, which features items and add-ons that people can use in the popular block-building game, has a new Education section.

Reviewer: ```suggestion
[Minecraft Marketplace](https://www.minecraft.net/marketplace), which features items and add-ons that people can use in the popular block-building game, has a new Education section.
```

The educational content lets players explore the International Space Station, through a partnership with NASA, as well as learn to code with a robot, visit Washington DC landmarks, find and build 3D fractals, learn what it’s like to be a marine biologist and more. This is launching for free download today and will be available through June 30, 2020. [Minecraft FREE Educational Session](https://news.microsoft.com/en-gb/2020/03/25/minecraft-launches-free-educational-content-for-children/)

> **NOTE: THE USER ACCOUNTS MAY NOT BE AVAILABLE PAST BUILD. ANY STUDENT WITH A M365 SCHOOLS ACCOUNT WILL BEABLE TO LOG INTO MINECRAFT EDUCATION**

Reviewer: ```suggestion

```

| COM235 | Thanks for joining the Student Zone at Build | [Chloe Condon](https://twitter.com/ChloeCondon), [Crazy Aunt Lindsey](https://www.thefablab.com/), [Liam McGregor](www.linkedin.com/in/liamjmcgregor/), [Morgan Dukes](https://www.linkedin.com/in/morgandukes3/), [Swetha Prabakaran](twitter.com/swethapraba) | [Microsoft the Musical](http://aka.ms/musical) |  |  |

## Try out Minecraft: Education Edition with MakeCode

You will need an Office 365 Education account to log in to the full version of Minecraft: Education Edition. If you do not have a license, you can still use Minecraft: Education Edition with a limited number of logins.

Enter your [school email address](https://education.minecraft.net/get-started/) to see if your school account is eligible for a trial. If it’s not, we’ll redirect you to a sign-up page.

### You can try out the Minecraft: Education Edition with these usernames:  
mee01@minecrafteducationedition.com  
mee02@minecrafteducationedition.com  
mee03@minecrafteducationedition.com  
...  
mee99@minecrafteducationedition.com  
Pick one, doesn't matter
Password for all: Build2020  
  

> **NOTE: THESE USER ACCOUNTS MAY NOT BE AVAILABLE PAST BUILD. HOWEVER, ANY STUDENT WITH A M365 SCHOOLS ACCOUNT WILL BE ABLE TO LOG INTO MINECRAFT EDUCATION**

Reviewer: ```suggestion
**NOTE: THESE USER ACCOUNTS MAY NOT BE AVAILABLE PAST BUILD. HOWEVER, ANY STUDENT WITH A M365 SCHOOLS ACCOUNT WILL BE ABLE TO LOG INTO MINECRAFT: EDUCATION EDITION**
```

### Minecraft Education 

-You can find more details of [Minecraft: Education Edition](https://education.minecraft.net/)
-Install [Minecraft: Education Edition](https://education.minecraft.net/get-started/)
  
### Educational Content on Minecraft
[Minecraft](https://www.minecraft.net/) has released a series of free educational sessions that parents can download to help children learn at home during COVID-19.

[Minecraft Marketplace](https://www.minecraft.net/marketplace), which features items and add-ons that people can use in the popular block-building game, has a new Education section.

The educational content lets players explore the International Space Station, through a partnership with NASA, as well as learn to code with a robot, visit Washington DC landmarks, find and build 3D fractals, learn what it’s like to be a marine biologist and more. This is launching for free download today and will be available through June 30, 2020. [Minecraft FREE Educational Session](https://news.microsoft.com/en-gb/2020/03/25/minecraft-launches-free-educational-content-for-children/)

> If these are no longer available, we recommend you check out the Minecraft: Education Edition Hour of Code, which is free and doesn't require a login!  

Reviewer: ```suggestion

```

| COM235 | Thanks for joining the Student Zone at Build | [Chloe Condon](https://twitter.com/ChloeCondon), [Crazy Aunt Lindsey](https://www.thefablab.com/), [Liam McGregor](www.linkedin.com/in/liamjmcgregor/), [Morgan Dukes](https://www.linkedin.com/in/morgandukes3/), [Swetha Prabakaran](twitter.com/swethapraba) | [Microsoft the Musical](http://aka.ms/musical) |  |  |

## Try out Minecraft: Education Edition with MakeCode

You will need an Office 365 Education account to log in to the full version of Minecraft: Education Edition. If you do not have a license, you can still use Minecraft: Education Edition with a limited number of logins.

Enter your [school email address](https://education.minecraft.net/get-started/) to see if your school account is eligible for a trial. If it’s not, we’ll redirect you to a sign-up page.

### You can try out the Minecraft: Education Edition with these usernames:  
mee01@minecrafteducationedition.com  
mee02@minecrafteducationedition.com  
mee03@minecrafteducationedition.com  
...  
mee99@minecrafteducationedition.com  
Pick one, doesn't matter
Password for all: Build2020  
  

> **NOTE: THESE USER ACCOUNTS MAY NOT BE AVAILABLE PAST BUILD. HOWEVER, ANY STUDENT WITH A M365 SCHOOLS ACCOUNT WILL BE ABLE TO LOG INTO MINECRAFT: EDUCATION EDITION**

Reviewer: ```suggestion
**NOTE: THESE USER ACCOUNTS MAY NOT BE AVAILABLE PAST BUILD. HOWEVER, ANY STUDENT WITH A M365 SCHOOLS ACCOUNT WILL BE ABLE TO LOG INTO MINECRAFT: EDUCATION EDITION**  

If these are no longer available, we recommend you check out the [Minecraft: Education Edition Hour of Code](https://education.minecraft.net/lessons/minecraft-hour-of-code/), which is free and doesn't require a login!  
```

| COM235 | Thanks for joining the Student Zone at Build | [Chloe Condon](https://twitter.com/ChloeCondon), [Crazy Aunt Lindsey](https://www.thefablab.com/), [Liam McGregor](www.linkedin.com/in/liamjmcgregor/), [Morgan Dukes](https://www.linkedin.com/in/morgandukes3/), [Swetha Prabakaran](twitter.com/swethapraba) | [Microsoft the Musical](http://aka.ms/musical) |  |  |

## Try out Minecraft: Education Edition with MakeCode

> You will need an Office 365 Education account to log in to the full version of Minecraft: Education Edition. If you do not have a license, you can still use Minecraft: Education Edition with a limited number of logins.

Reviewer: ```suggestion
You will need an Office 365 Education account to log in to the full version of [Minecraft: Education Edition](https://education.minecraft.net/). If you do not have a license, you can still use Minecraft: Education Edition with a limited number of logins that we've provide below.
```

> Enter your [school email address](https://education.minecraft.net/get-started/) to see if your school account is eligible for a trial. If it’s not, we’ll redirect you to a sign-up page.

Reviewer: ```suggestion
To get started, enter your [school email address on the Minecraft: Education Edition website](https://education.minecraft.net/get-started/) to confirm that your school account is eligible for full access or even a free trial. Then, download the game and follow the sign in prompts from within the game, using your same school credentials.  

Additional support can be found on the [Help page for Minecraft: Education Edition](https://minecrafteducation.zendesk.com/). 
```

| COM235 | Thanks for joining the Student Zone at Build | [Chloe Condon](https://twitter.com/ChloeCondon), [Crazy Aunt Lindsey](https://www.thefablab.com/), [Liam McGregor](www.linkedin.com/in/liamjmcgregor/), [Morgan Dukes](https://www.linkedin.com/in/morgandukes3/), [Swetha Prabakaran](twitter.com/swethapraba) | [Microsoft the Musical](http://aka.ms/musical) |  |  |

## Try out Minecraft: Education Edition with MakeCode

You will need an Office 365 Education account to log in to the full version of [Minecraft: Education Edition](https://education.minecraft.net/). If you do not have a license, you can still use Minecraft: Education Edition with a limited number of logins that we've provide below.

To get started, enter your [school email address on the Minecraft: Education Edition website](https://education.minecraft.net/get-started/) to confirm that your school account is eligible for full access or even a free trial. Then, download the game and follow the sign in prompts from within the game, using your same school credentials.  

Additional support can be found on the [Help page for Minecraft: Education Edition](https://minecrafteducation.zendesk.com/). 

> ### You can try out the Minecraft: Education Edition with these usernames:  

Reviewer: ```suggestion
### Demo accounts for Minecraft: Education Edition
If you do not have access to Minecraft: Education Edition through your school, you can try it out with these limited supply and limited time usernames:  
```
